groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -116,6 +131,61 @@ public Path getWorkspaceRoot() {
     }
 
     @Override
+    public Map<String, Set<SymbolInformation>> getReferences() {
+        return references;
```

Comment:
@gracew I think it's a reasonable behavior here. Since even in intelliJ, you won't be able to find all the references without a full build.
@natgabb I remember you told me the reason that you are not calling `compile()` first in `getFileSymbols` is `parseAllSymbols` will take a long time. Looks like it's the same case here. But do you think it's good if we do compile once during initialization? Or should we defer it to the code editor to make the call explicitly